
<launch>

  <arg name="limb"     default="right"/>
  <arg name="method"   default="color"/>

  <param name="robot_description" textfile="$(find baxter_description)/urdf/baxter.urdf"/>

  <param name="/object_tracker/config_folder" value="$(find baxter_demos)/config/"/>

  <node pkg="baxter_demos" name="object_finder_3d" type="object_finder.py"
  args="--limb $(arg limb) --method $(arg method) --topic /camera/rgb/image_rect_color" required="true">
  </node>

  <node pkg="baxter_demos" name="object_finder" type="object_finder.py"
  args="--limb $(arg limb) --method $(arg method) --topic /cameras/right_hand_camera/image" required="true">
  </node>
  
  <node pkg="baxter_demos" name="get_goal_poses" type="get_goal_poses.py"
  args="--limb $(arg limb)">
  </node>

  <node pkg="baxter_demos" name="estimate_depth" type="estimate_depth.py"
  args="--limb $(arg limb)">
  </node>

  <node pkg="baxter_demos" name="servo_to_object" type="super_stacker.py"
args="--limb $(arg limb)" required="true">
    </node>


</launch>
